util/boac_utils.rb

```diff
@@ -260,4 +271,35 @@ def self.get_dismissed_alerts(alerts)
     end
   end
 
+  # Deletes the dismissal of an alert by a given user or the admin test user by default
+  # @param alert [Alert]
+  # @param user [User]
+  def self.remove_alert_dismissal(alert, user = nil)
+    query = "DELETE
+              FROM alert_views
+              WHERE alert_views.alert_id = '#{alert.id}'
```

Comment:
@pfarestveit I woud expect postgres to be unhappy with the quotes above since `alert_id` is an integer column.  I suppose postgres is tolerant.